src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSFeatureConfig.java

```diff
@@ -16,9 +16,4 @@
 
 import com.google.api.codegen.transformer.FeatureConfig;
 
-public class NodeJSFeatureConfig extends FeatureConfig {
-  @Override
-  public boolean enableGrpcStreaming() {
-    return true;
-  }
-}
+public class NodeJSFeatureConfig extends FeatureConfig {}
```

Comment:
We can rename `FeatureConfig` to `DefaultFeatureConfig` so the languages use default values will not need to create an empty subclass.